OpenWebNet automatic scan of automation, lighting and temperature control devices.

Spugna85 opened the pull request:
Implementation of automatic scan of automation, lighting and temperature
control.
This include automatic insert of any automation, lighting and temperature control devices.

Other bug fix....

Spugna85 committed:
Automatic scan of automation, lighting and temperature control

Implementation of automatic scan of automation, lighting and temperature
control.
This include automatic insert of any device of automation, lighting and
temperature control.

gizmocuz commented on main/RFXNames.cpp:
```diff
@@ -498,6 +498,7 @@ const char *RFX_Type_SubType_Desc(const unsigned char dType, const unsigned char
 		{ pTypeTEMP, sTypeTEMP9, "RUBiCSON" },
 		{ pTypeTEMP, sTypeTEMP10, "TFA 30.3133" },
 		{ pTypeTEMP, sTypeTEMP11, "WT0122 Pool sensor" },
+		{ pTypeTEMP, sTypeTEMP12, "Legrand MyHome Temp" },
```
You can not do this, just use any of the existing temp sensors, no need for a 12th
this is only for rfxcom, please use
pTypeGeneral/sTypeTemperature

gizmocuz commented on main/RFXtrx.h:
```diff
@@ -8,11 +8,11 @@
 
 /*
-                                                                   
+
```
You can not touch this file, it is not ours

gizmocuz commented on main/RFXtrx.h:
```diff
@@ -1673,7 +1674,7 @@ typedef union tRBUF {
 #endif
 		BYTE	temperaturel;
-		BYTE	humidity; 
+		BYTE	humidity;
```
Also turn of the formatting option in your editor

gizmocuz commented on www/app/LightsController.js:
```diff
@@ -3482,9 +3482,17 @@ define(['app'], function (app) {
-			else if ((lighttype==305) || (lighttype==306)) {
-				//OpenWebNet Blinds/light
-				var ID="OpenWebNet";
+			else if (lighttype==305) {
```
you test for 305 now only, in lights.html you removed 305 and 306, is this going OK?

Spugna85 commented:
can you merge this? is there any problem in this request?

gizmocuz commented:
Maybe I asked a question above?

Spugna85 committed:
Restore some files and fix add device openwebnet manually

- Restoring RFXNames.cpp and RFXtrx.h file
- fix of adding the device manually

Spugna85 commented:
I fixed all in the second commit.

Spugna85 committed:
OpenWebNet fix crash with who=13

who=13 (WHO_GATEWAY_INTERFACES_MANAGEMENT) is not yet supported, but is
now managed and does not cause a system crash.

gizmocuz commented:
Thanks!
